backend/app/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class AppUser(models.Model):
+  user = models.OneToOneField(User, on_delete=models.CASCADE)
+  avatar_url = models.CharField(max_length=100, blank=True)
```

Comment:
i think there is a `UrlField` for url data 
- https://docs.djangoproject.com/en/4.1/ref/models/fields/#urlfield